### ISSUE TITLE
Unable to disable constraints on a table with indexes

### ISSUE THREAD

**Issue by tomsquest** [tomsquest/datability#1]
Given the table

```
CREATE TABLE myTable
(
  id INTEGER NOT NULL
);
CREATE INDEX myIndex ON myTable (id);
```

When calling `datability.DatabaseException: Error thrown executing sql 'ALTER TABLE myTable DROP CONSTRAINT myIndex'. Message: ERROR: constraint "myIndex" of relation "myTable" does not exist` is returned

Analysis : seems that indexes are seen as constraint, and then Datability tries to `drop constraint` on them.

**Comment by tomsquest** [comment]
Fixed by bea98ecd1829df6f9da2069bb36c0e09a0d482e1

**Comment by tomsquest** [comment]
Fixed by bea98ecd1829df6f9da2069bb36c0e09a0d482e1
